extension/src/Xliff/XLIFFDocument.ts

```diff
@@ -190,7 +195,15 @@ export class Xliff implements XliffDocumentInterface {
     if (formatXml) {
       xml = Xliff.formatXml(xml, this.lineEnding);
     }
-
+    for (let index = 0; index < searchReplaceBeforeSaveXliff.length; index++) {
+      const element = searchReplaceBeforeSaveXliff[index];
+      if (
+        element.replaceWith !== undefined &&
+        element.searchFor !== undefined
+      ) {
+        xml = Common.replaceAll(xml, element.searchFor, element.replaceWith);
+      }
+    }
```

Comment:
Not a strong objection but a small reflection; Is this a feature of `Xliff` or the extension? Kinda feels like we're mixing in an extension feature here. Although we've probably done that in one way or another :sweat_smile: this might affect future revisions of the feature.

Reply:
I can clearly see this as an feature of `Xliff`

Reply:
At least it unlocks all kinds of different wishes for the Xliff file formatting :)

Reply:
The simplest solution I could think of to work around this would be to add a `callback` param

Reply:
I cannot do this right now, I cannot find a way to do this with a reasonable amount of effort. 😉

Reply:
Same here hopefully I'll get some more time to spare within a few weeks :+1: